app/assets/javascripts/profile/email.coffee

```diff
@@ -26,6 +26,7 @@ class Email
     ev.preventDefault()
     $.ajax({type: "PUT", url: @url('resend_confirmation')})
       .success( (resp) =>
+        ev.target.disabled = true
         OX.Alert.display(message: resp.message, type: 'success', parentEl: @$el)
```

Comment:
Thanks for your comment, Nathan.
Reason why Dante and I suspect that it's bots following the URL is because we were not able to reproduce the error by double clicking ourselves manually or with RSpec (`double_click`).  To abuse that button, we suspect it requires an async request; we tried creating multiple `Threads` but that didn't work with RSpec. So we agreed that I should use `button_to` instead of `link_to` since it "is the safest method to ensure links that cause changes to your data are not triggered by search bots or accelerators"—and also disable it on click.

I'll make the fixes that you recommended.